auto_install/install.sh

```diff
@@ -2597,7 +2825,13 @@ All incomplete posts or bug reports will be ignored or deleted.\\n\\nThank you f
 		whiptail --title "Rebooting" --msgbox "The system will now reboot." ${r} ${c}
 		printf "\\nRebooting system...\\n"
 		$SUDO sleep 3
-		$SUDO shutdown -r now
+
+		if [ "${PLAT}" == 'Alpine' ]
+		then
+			"${SUDO}" reboot
+		else
+			"${SUDO}" shutdown -r now
+		fi
```

Comment:
Same as at the other place, `reboot` should always work, systemd ships it on all Debian variants and based distros with the `systemd-sysv` package that is mandatory to have systemd the init system:
```suggestion
		$SUDO reboot
```